hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -184,15 +186,19 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
           // Differentiate key without trailing slash
           return (subpath.isKey || subpathName === OTHER_PATH_NAME) ? subpathName : subpathName + '/';
         });
-
+         
         values = subpaths.map(subpath => {
           return subpath.size / dataSize;
         });
-
+        // Logic for Values Normalization on Pie chart adding 0.05 so minimum block
+        // will be displayed on Pie chart using below logic
+        // Percentage and Size string Logic will be same 
+        normalizedValues= normalize(values);
+        normalizedValues= values && values.map((item)=> item+ 0.05);
         percentage = values.map(value => {
           return (value * 100).toFixed(2);
         });
-
+       
```

Comment:
Please remove this change.

Reply:
@ArafatKhan2198 
Done changes in latest commit